frontend/src/js/external-forms/form-components/DropzoneBetweenElements.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
+import { useState } from "react";
+import { DropTargetMonitor } from "react-dnd";
+
+import Dropzone, {
+  PossibleDroppableObject,
+} from "../../ui-components/Dropzone";
+
+interface Props {
+  onDrop: (props: PossibleDroppableObject, monitor: DropTargetMonitor) => void;
+  acceptedDropTypes: string[];
+  top: number;
+  height: number;
+}
+
+const LineHeight = 3;
+
+const Line = styled("div")<{ show: boolean }>`
+  overflow: hidden;
+  display: block;
+  visibility: ${({ show }) => (show ? "visible" : "hidden")};
+  background-color: ${({ theme }) => theme.col.blueGrayDark};
+  margin: 1px 0;
+  height: ${LineHeight}px;
```

Comment:
(we can make this as big as `5px` I think)

Reply:
I think thats a little about personal preference. I think 5 px is a little fat for the line -> I would prefer 3px then 4px. I will push it with 4 let me know what you think